### PR TITLE
integration of text into window

### PR DESCRIPTION
**Pull request by TalPat**
Text can be rendered onto the screen using the writeLine function of the render class (only 'space' and alpha-numeric characters currently)

### CODE REVIEW

**Review by MEWestvig — APPROVED** [review]

File: app/src/Bomberman.cpp [hunk 1]
```diff
@@ -58,6 +58,7 @@ void Bomberman::updateFunc()
 	if (this->frameClock.getElapsedTime().asSeconds() >= this->perFrameSeconds)
 	{
 		this->renderer.render(*(this->window), this->gameState);
+		renderer.writeLine(*window, "FPS "+std::to_string((int)(1/this->frameClock.getElapsedTime().asSeconds())), sf::Vector3i(10,20,50), sf::Vector2f(-1.0f,-1.0f), 0.2f); // example of implementation, rather use in renderer to avoid syncopated draw calls
```
Comment (GJSteyn) [inline app/src/Bomberman.cpp]: Spacing on this line. If it's too long, consider splitting.

File: renderer/CMakeLists.txt [hunk 2]
```diff
@@ -9,7 +9,8 @@ add_library(Renderer STATIC
     src/Model.cpp
     src/Shader.cpp
     src/Mesh.cpp
-    src/Animation.cpp
+		src/Animation.cpp
+		src/GuiChar.cpp
 )
```
Comment (GJSteyn) [inline renderer/CMakeLists.txt:15]: Spaces instead of tabs... Yes, the one place that spaces are preferred

File: renderer/src/GuiChar.cpp [hunk 3]
```diff
@@ -0,0 +1,95 @@
+#include "GuiChar.hpp"
+#include "stb_image.h"
+
+GuiChar::GuiChar(std::string path)
+{
+	std::vector<float> vertices = {
+			0.5f, 0.5f, 0.0f, 1.0f, 0.0f, //tr
+			0.5f, 0.0f, 0.0f, 1.0f, 1.0f, //br
+			0.0f, 0.0f, 0.0f, 0.0f, 1.0f, //bl
+			0.0f, 0.5f, 0.0f, 0.0f, 0.0f	//tl
```
Comment (GJSteyn) [inline renderer/src/GuiChar.cpp]: Just something stupid, but could we get the comment indentation consistent here?

File: renderer/src/GuiChar.cpp [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+	};
+	std::vector<unsigned int> indices = {
+			0, 1, 3,
+			1, 2, 3};
```
Comment (GJSteyn) [inline renderer/src/GuiChar.cpp]: This closing brace should be on the next line.

File: renderer/src/GuiChar.cpp [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+	textureId = textureFromFile(path.c_str());
+}
+
+GuiChar::~GuiChar()
+{
+}
+
+void GuiChar::draw(Shader shader)
+{
+
```
Comment (GJSteyn) [inline renderer/src/GuiChar.cpp]: Extra line.

File: renderer/src/GuiChar.hpp [hunk 6]
```diff
@@ -0,0 +1,23 @@
+#ifndef GUICHAR_HPP
+#define GUICHAR_HPP
+
+#include <glm/glm.hpp>
+#include "Shader.hpp"
+#include <vector>
+#include <map>
+
+
```
Comment (GJSteyn) [inline renderer/src/GuiChar.hpp]: Extra line

**Review by TalPat — COMMENTED** [review]
Formatting issues have been addressed. I hope its a bit easier on the eyes now.

**Review by GJSteyn — APPROVED** [review]

**Pull request merged**